Lib/pathlib.py

```diff
@@ -751,7 +694,7 @@ def cwd(cls):
         """Return a new path pointing to the current working directory
         (as returned by os.getcwd()).
         """
-        return cls(os.getcwd())
+        return cls().absolute()
```

Comment:
Why this change?

Reply:
Technically not necessary for this patch, but it aligns `cwd()` and `absolute()` with `home()` and `expanduser()`, i.e. the classmethod calls through to the instance method, and not vice-versa.

This should give a small performance improvement in `path.absolute()`, which I suspect is used much more than `Path.cwd()`.

Reply:
I could move this to another PR if you thought it appropriate?

Reply:
The previous implementation was straightforward and matched exactly what the docstring said.
Now it is more indirect and it’s not obvious that it does the exact same thing.

I feel that «aligning» is not a strong reason to change here (and I don’t understand the comment about helping performance of `absolute`).

Reply:
I'll undo the change as I'm failing to easily justify it :-).

I'll log it as a separate (minor) performance bug with a fuller explanation.

Reply:
FYI, I logged this as #100562